seed/static/seed/js/controllers/mapping_controller.js

```diff
@@ -189,7 +194,7 @@ angular.module('BE.seed.controller.mapping', [])
           resolve: {
             action: _.constant('new'),
             data: _.constant({mappings: preset_mapping_data, preset_type: presetType}),
-            org_id: _.constant(user_service.get_organization().id)
+            org_id: _.constant($scope.organization.id)
```

Comment:
> Made a similar change to the cross-cycles tab since the fix looked similar. After that, I discovered the inventory detail page and inventory detail cycles page don't use org display sig fig settings. The slope is feeling a little slippery 😬 but maybe those two pages are the last ones... Think you could look into those two?

My impression is that the detail cycles page would be a fair amount of work because it's not directly using the ui-grid api, but making tables in the partial. So it's easy to get it formatted correctly, but then the sorting will be broken (and I don't know how to do that when using html rather than the ui-grid api).

I was able to update the display on the details page since there's no sorting

Reply:
Oh I was referring to the inventory detail cross-cycles tab `inventory_detail_cycles.html`, which I think should be similar to the inventory detail page (in terms of how a fix might be implemented).